README.md

```diff
@@ -1,2 +1,105 @@
+- Case WDL: Calling case samples using a previously built model for denoising: ``cnv_germline_case_workflow.wdl``
+- Scattered case WDL (recommended): Functionally equivalent to case WDL, written for reducing cloud compute cost (see below) and wall-clock time ``cnv_germline_case_scattered_workflow.wdl``
+
+
+
+#### Setting up parameter json file for a run
+
+To get started, create the json template (using ``java -jar wdltool.jar inputs <workflow>``) for the workflow you wish to run and adjust parameters accordingly.
+
+*Please note that there are optional workflow-level and task-level parameters that do not appear in the template file.  These are set to reasonable values by default, but can also be adjusted if desired.*
+
+#### Required parameters in the germline cohort workflow
+
+The reference used must be the same between PoN and case samples.
+
+- ``CNVGermlineCohortWorkflow.cohort_entity_id`` -- Name of the cohort.  Will be used as a prefix for output filenames.
+- ``CNVGermlineCohortWorkflow.contig_ploidy_priors`` -- TSV file containing prior probabilities for the ploidy of each contig, with column headers: CONTIG_NAME, PLOIDY_PRIOR_0, PLOIDY_PRIOR_1, ...
+- ``CNVGermlineCohortWorkflow.gatk_docker`` -- GATK Docker image (e.g., ``broadinstitute/gatk:latest``).
+- ``CNVGermlineCohortWorkflow.intervals`` -- Picard or GATK-style interval list.  For WGS, this should typically only include the chromosomes of interest.
+- ``CNVGermlineCohortWorkflow.normal_bais`` -- List of BAI files.  This list must correspond to `normal_bams`.  For example, `["Sample1.bai", "Sample2.bai"]`.
+- ``CNVGermlineCohortWorkflow.normal_bams`` -- List of BAM files.  This list must correspond to `normal_bais`.  For example, `["Sample1.bam", "Sample2.bam"]`.
+- ``CNVGermlineCohortWorkflow.num_intervals_per_scatter`` -- Number of intervals (i.e., targets or bins) in each scatter for GermlineCNVCaller.  If total number of intervals is not divisible by the value provided, the last scatter will contain the remainder.
+- ``CNVGermlineCohortWorkflow.ref_fasta_dict`` -- Path to reference dict file.
+- ``CNVGermlineCohortWorkflow.ref_fasta_fai`` -- Path to reference fasta fai file.
+- ``CNVGermlineCohortWorkflow.ref_fasta`` -- Path to reference fasta file.
+- ``CNVGermlineCohortWorkflow.maximum_number_events_per_sample`` -- Maximum number of events threshold for doing sample QC (recommended for WES is ~100)
+
+In additional, there are optional workflow-level and task-level parameters that may be set by advanced users; for example:
+
+- ``CNVGermlineCohortWorkflow.do_explicit_gc_correction`` -- (optional) If true, perform explicit GC-bias correction when creating PoN and in subsequent denoising of case samples.  If false, rely on PCA-based denoising to correct for GC bias.
+- ``CNVGermlineCohortWorkflow.PreprocessIntervals.bin_length`` -- Size of bins (in bp) for coverage collection.  *This must be the same value used for all case samples.*
```

Comment:
Is this just for WGS?

Reply:
According to the [tutorial](https://gatk.broadinstitute.org/hc/en-us/articles/360035531152--How-to-Call-common-and-rare-germline-copy-number-variants#1) it should be set to 1000 for WGS and 0 for exomes. I noticed the default value for the task that uses the variable (PreprocessIntervals) has a [default value](https://github.com/gatk-workflows/gatk4-germline-cnvs/blob/11f1870ffe202d0687486e267213a9634421f0c3/tasks/cnv_common_tasks.wdl#L36) for the variable of 1000. Since this workflow is directed towards exomes should the input json have this variable set to 0? @samuelklee

Reply:
You can technically use it for WES as well (to break targets up into smaller bins), we just haven't evaluated it.  Let's leave the documentation unchanged, but set the default to 0 in the JSON---thanks for catching that, @bshifaw!